plugins/operators/__init__.py

```diff
@@ -915,28 +901,33 @@ def _delete_sample_field_inputs(ctx, inputs):
         prop.invalid = True
         return
 
-    field_keys = list(schema.keys())
-    field_selector = types.AutocompleteView()
-    for key in field_keys:
-        field_selector.add_choice(key, label=key)
+    field_names = _to_string_list(ctx.params.get("field_names", []))
 
-    field_prop = inputs.enum(
-        "field_name",
-        field_selector.values(),
+    field_choices = types.AutocompleteView(multiple=True)
+    for key in schema.keys():
+        if not any(key == f or key.startswith(f + ".") for f in field_names):
+            field_choices.add_choice(key, label=key)
+
+    field_prop = inputs.list(
+        "field_names",
+        types.OneOf([types.Object(), types.String()]),
```

Comment:
Ideally can make this just `types.String()` and remove `_to_string_list()` usage